core/sass/high-potential-opportunity-detail.scss

```diff
@@ -198,12 +197,10 @@
   }
   ul {
     margin-top: 70px;
-    li.company-list-first-line {
+    column-count: 4;
```

Comment:
`column-count` is not compatible with IE <9. will that be an issue?

Reply:
the outcome is acceptable: the items will be shown in a horizontal list

Reply:
sounds fair 👍